admin_manual/appliance/installation.rst

```diff
@@ -1,26 +1,25 @@
-=================
-How to Install It
-=================
+============================
+How to Install the Appliance
+============================
 
 The install process is a little involved, but not too much. 
 To keep it succinct, you need to:
 
-- :ref:`Download <appliance-download-label>` and :ref:`install <appliance-download-label>` the appliance 
-- Step through :ref:`the configuration wizard <appliance-download-label>`
-- :ref:`Activate <appliance-download-label>` the configured appliance 
+- :ref:`Download <appliance-download-label>` and :ref:`Install <appliance-install-label>` the appliance 
+- Step through :ref:`the configuration wizard <appliance-start-label>`
+- :ref:`Activate <appliance-activate-label>` the configured appliance 
 
-After that, you can access the running instance of ownCloud and :ref:`further configure it <appliance-download-label>` to suit your needs. 
+.. important:: 
+   You need **Internet access** to use the appliance. The appliance **has to be activated** with a license you get from Univention via **mail**. This license has to be imported in the appliance via the **web interface**.
```

Comment:
I'd suggest the following change here: 

s/The appliance **has to be activated** with a license you get from Univention via **mail**./The appliance **has to be activated** with a license that you will receive from Univention via email.